### DISCUSSION TITLE
InstantSearch

### DISCUSSION THREAD

**Discussion by rrg92** [requarks/wiki#5900] [General]
Please, consider use of InstantSearch to build searching UI and allow us take more control over searching UI https://www.algolia.com/doc/guides/building-search-ui/what-is-instantsearch/js/

**Comment by NGPixel** [comment]
Use https://requarks.canny.io/wiki to submit feature requests.

However note that external search engines are being deprecated in v3.

**Reply by rrg92** [reply]
Sorry. I will move to there. Thanks answer.

Other question: You will remove the elastic search support?

**Reply by NGPixel** [reply]
Yes. Because v3 will only support PostgreSQL, it allows for a robust internal search engine to be used rather than relying on external ones.
